caravel/assets/javascripts/SqlLab/components/App.jsx

```diff
@@ -37,16 +39,18 @@ class App extends React.Component {
         </div>
       );
     }
+    content = (
```

Comment:
I realized content here gets reassigned when this.state.hash is true, so QuerySearch page is not shown, is this intended? @mistercrunch